src/Components/ApplicationCard/index.js

```diff
@@ -5,7 +5,7 @@ import Icon from '../Icon';
 
 function ApplicationCard(props) {
   const {
-    type, userCount, approvedCount, onClick,
+    type, userCount, approvedCount,
```

Comment:
these could be destructured in the parens for this function component ( i.e. `ApplicationCard({type, user.. etc.})` )
but obv no big at all. either way looks good ✔️